### DISCUSSION TITLE
Parameter optimisation error message

### DISCUSSION THREAD

**Discussion by bahsa505** [Open-Systems-Pharmacology/Forum#1204] [Questions & Problems]
Hi all

I have been getting this error message when trying to run parameter identification while considering two datasets, but not when using each dataset separately.

![image](https://user-images.githubusercontent.com/61779799/164486127-c3f34abb-a57f-47dc-8598-d866ef15dab9.png)

Any thoughts what is the problem?

**Comment by msevestre** [comment]
Can you attach your project so that we can reproduce?
Also what do you mean exactly with "considering two datasets"?

**Reply by bahsa505** [reply]
I mean trying to optimise the model parameter while fitting the model to more than one data source (either individual data from different subjects, or different simulations on different dosing regimens/ publications) 

I am getting this error when I follow the documentation on parameter identification for the simple case with theophylline. I am trying to upload the file, but it says it is not supported, but it's the theophylline example file.

![image](https://user-images.githubusercontent.com/61779799/164728352-adb618c4-9e07-4739-bf38-13df5b81e560.png)

https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#results-of-parameter-identification

Seems to happen if I asked for 'show visual feedback' simultaneously while running the optimisation. It did not happen though if I only run the optimisation, or was only considering one data source! 

Not sure if this is a bug or a problem with my installation

**Reply by msevestre** [reply]
it's a bug. When you get this message, it's definitely a bug
Can you attach your project please and list the steps to reproduce the issue?

**Reply by msevestre** [reply]
please zip the file and then upload

**Reply by bahsa505** [reply]
Thanks for the update, here is the model file 

[Theophylline.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/8541749/Theophylline.zip)


For the steps I just followed documentation page;
1.In the Simulation Explorer mark the two simulations "Kaumeier IV 208 mg fit" and "Kaumeier oral solution 185mg" and select  Start Parameter Identification ... from the context menu.
2. A new view for Parameter Identification 1 is displayed and the tab Data is opened. On the left, the simulations with the assigned observed data are displayed. On the right, a list of mappings from outputs to observed data is displayed. For each observed data set for concentrations and fractions the corresponding output is mapped automatically (based on the Organ, Compartment, Molecule meta data).
3. In the tab Configuration keep the default settings. In the Ribbon Bar Run & Analyze click Show Visual Feedback and then Run to start the optimization.
5. Now the Visual Feedback Window shows the intermediate state during the Parameter Identification Run.
6. After some iterations, the run is finished and you can switch to the tab Results.
